src/main/kotlin/org/rust/ide/refactoring/RsImportOptimizer.kt

```diff
@@ -39,7 +39,7 @@ class RsImportOptimizer : ImportOptimizer {
         externCrateItems.forEach { it.delete() }
     }
 
-    private fun executeForUseItem(mod: RsMod) {
+    fun executeForUseItem(mod: RsMod) {
```

Comment:
[minor] Now it may be better to introduce public `executeForUseItem` before private `executeForExternCrate`

Reply:
I think it is better to keep as-is, so that `executeForUseItem` and `optimizeUseSpeck` functions will be close to each other